app/TuistApp/Sources/Views/SimulatorsView/SimulatorsViewModel.swift

```diff
@@ -0,0 +1,188 @@
+import Foundation
+import PathKit
+import SwiftUI
+import TuistAutomation
+import TuistCore
+import TuistServer
+import TuistSupport
+import XcodeGraph
+
+enum SimulatorsViewModelError: FatalError, Equatable {
```

Comment:
`FatalError` was designed for the CLI, so I wonder if it makes more sense here to use a plain `Error` that conforms to `CustomStringConvertible` and `Equatable` instead:

```suggestion
enum SimulatorsViewModelError: Error, Equatable {
```

Reply:
I don't see `FatalError` as CLI specific. Also, since we are reusing libraries like `TuistSupport`, we need to handle `FatalError` regardless. I'd rather keep the same convention here.